lib/responses.js

```diff
@@ -218,7 +231,12 @@ function denyReferenceAtRoot(req, res, next) {
   const body = req.body;
 
   if (_.has(body, '_ref')) {
-    sendDefaultResponseForCode(400, 'Reference (_ref) at root of object is not acceptable', res);
+    const code = 400,
+      message = 'Reference (_ref) at root of object is not acceptable';
+
+    log('error', message, { code });
+
+    sendDefaultResponseForCode(code, message, res);
```

Comment:
(see +171 to +173)

Reply:
the context for the 400 from this function and the one on line 214 are different so it would be worth logging it